vendor/k8s.io/kube-openapi/pkg/handler3/handler.go

```diff
@@ -49,9 +52,20 @@ const (
 	subTypeJSON     = "json"
 )
 
+// OpenAPIV3Discovery is the format of the Discovery document for OpenAPI V3
+// It maps Discovery paths to their corresponding URLs with a hash parameter included
+type OpenAPIV3Discovery struct {
+	Paths map[string]OpenAPIV3DiscoveryGroupVersion
+}
+
+// OpenAPIV3DiscoveryGroupVersion includes information about a group version and URL
+// for accessing the OpenAPI. The URL includes a hash parameter to support client side caching
+type OpenAPIV3DiscoveryGroupVersion struct {
+	URL string
```

Comment:
is this a full URL (including hostname/port) or a relative path? if a path, is it a request-relative path or an absolute path?

Reply:
Any suggestions on naming? I've opened https://github.com/kubernetes/kube-openapi/pull/295

Reply:
`Path` seems most accurate to me

```
Path string `json:"path"`
```

Reply:
actually, I guess it includes a query param, so it's not *just* a path :-/